packages/web3-utils/test/unit/socket_provider.test.ts

```diff
@@ -33,13 +33,17 @@ class TestProvider extends SocketProvider<any, any, any> {
 	// eslint-disable-next-line
 	protected _removeSocketListeners(): void {}
 	// eslint-disable-next-line
-	protected _onCloseEvent(_event: unknown): void {}
+	protected _onCloseEvent(_event: any): void {}
```

Comment:
Any reason to replace `unknown`? I think it's a better approach than using `any`